t/integration/test_canvas.py

```diff
@@ -1497,11 +1498,14 @@ def test_chord_on_error(self, manager):
         if not manager.app.conf.result_backend.startswith('redis'):
             raise pytest.skip('Requires redis result backend.')
 
-        # Run the chord and wait for the error callback to finish.
+        # Run the chord and wait for the error callback to finish. Note that
+        # this only works for old style callbacks since they get dispatched to
+        # run async while new style errbacks are called synchronously so that
+        # they can be passed the request object for the failing task.
         c1 = chord(
             header=[add.s(1, 2), add.s(3, 4), fail.s()],
             body=print_unicode.s('This should not be called').on_error(
-                chord_error.s()),
```

Comment:
Oh, the `chord_error` test task is unused now I think. I can probably remove it prior to merge, just forgot to do so when I pushed this diff.

Reply:
ok

Reply:
done